src/components/pages/ManageNodePage.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useMemo } from "react";
+import { useSelector } from "react-redux";
+import type { ColumnDef } from "@tanstack/react-table";
+
+import type { NodeInfo } from "@bindings/protobufs/NodeInfo";
+import TableLayout from "@components/Table/TableLayout";
+import { selectAllNodes } from "@features/device/deviceSelectors";
+
+const ManageNodePage = () => {
+  const nodes = useSelector(selectAllNodes()).map((n) => n.data);
+  const columns = useMemo<ColumnDef<NodeInfo, unknown>[]>(
+    () => [
+      { header: "ID", accessorKey: "num" },
+      {
+        id: "User Name",
+        accessorFn: (n) => n.user?.shortName ?? "No user info",
+      },
+      {
+        id: "Last Heard",
+        accessorFn: (n) => (!n.lastHeard ? n.lastHeard : "No data"),
+      },
+      {
+        id: "Latitude",
+        accessorFn: (n) => n.position?.latitudeI ?? "No GPS lock",
+      },
+      {
+        id: "Longitude",
+        accessorFn: (n) => n.position?.longitudeI ?? "No GPS lock",
+      },
+      {
+        id: "Battery",
+        accessorFn: (n) =>
+          n.deviceMetrics?.batteryLevel && n.deviceMetrics.batteryLevel
+            ? `${n.deviceMetrics.voltage}V (${
+                n.deviceMetrics.batteryLevel * 100
```

Comment:
Not blocking, but I don't completely understand the calculation here and I don't think it's right:

![image](https://user-images.githubusercontent.com/57198797/221393941-3e74ecda-c3a1-4b5f-9fd1-fcc7de6bb2e7.png)

Reply:
Similarly, is location being set as a default of 0?

Reply:
Hmm interesting, haven't seen that one! Might have multiplied by one too many scaling factors on the percentage

Reply:
@sherry1iu Should be fixed! There was an extra 100x multiplier